src/app/screens/edit-personal-info/index.tsx

```diff
@@ -0,0 +1,158 @@
+import React, { useEffect, useState } from 'react';
+import { SubmitHandler, useForm } from 'react-hook-form';
+import { useTranslation } from 'react-i18next';
+import {
+  Alert,
+  KeyboardAvoidingView,
+  Platform,
+  SafeAreaView,
+  ScrollView,
+} from 'react-native';
+import { Box, ButtonText, Text } from '@gluestack-ui/themed';
+import { Button } from '@gluestack-ui/themed';
+import { yupResolver } from '@hookform/resolvers/yup';
+
+import { ControlledInput } from 'src/app/components/inputs';
+import ControlledSelectInput from 'src/app/components/inputs/select';
+import { ProfileData } from 'src/interfaces/profile-data';
+import profileDataFile from 'src/mocks/profile-data.json';
+import { commonStyles } from 'src/utils/styles';
+
+import { FormData, validationSchema } from './form-config';
+import { EditPersonalInfoProps } from './types';
+
+export const EditPersonalInfoScreen = ({
+  navigation,
+}: EditPersonalInfoProps) => {
+  const [profileData, setProfileData] = useState<ProfileData | null>(null);
+  const { t } = useTranslation();
+
+  const genderOptions = [
+    { label: t('common:gender.male'), value: 'MALE' },
+    { label: t('common:gender.female'), value: 'FEMALE' },
+  ];
+
+  const { control, handleSubmit, reset } = useForm<FormData>({
+    defaultValues: {
+      firstName: profileData?.name,
+      lastName: profileData?.birthDate,
+      birthDate: profileData?.birthDate,
+      gender: profileData?.gender,
+    },
+    resolver: yupResolver(validationSchema),
+  });
```

Comment:
Here you can call the formConfig object.
```suggestion
  const { control, handleSubmit, reset } = useForm<FormData>({
    ...formConfig,
    defaultValues: {
      firstName: profileData?.name,
      lastName: profileData?.birthDate,
      birthDate: profileData?.birthDate,
      gender: profileData?.gender,
    },
  });
```